crates/bevy_pbr/src/render/light.rs

```diff
@@ -600,8 +593,17 @@ pub fn prepare_lights(
 
     let mut point_lights: Vec<_> = point_lights.iter().collect::<Vec<_>>();
 
+    #[cfg(not(feature = "webgl"))]
+    let max_point_light_shadow_maps = point_lights
+        .iter()
+        .filter(|light| light.1.shadows_enabled)
+        .count()
+        .min((render_device.limits().max_texture_array_layers / 6) as usize);
```

Comment:
What's the 6 here and on line 624? I'm not sure what it's supposed to mean

Reply:
True. It is a ‘magic number’ I guess though it’s already been used elsewhere so I’d fix it in a separate PR. Point lights use cubemaps for shadow mapping. Cubemaps are array textures with 6 layers per cubemap, one for each cube face. That’s where the 6 comes from.